packages/main/src/RPA/Browser/Selenium.py

```diff
@@ -1949,3 +1950,19 @@ def _send_command_and_get_result(self, cmd, params):
         response = self.driver.command_executor._request("POST", url, body)
 
         return response.get("value")
+
+    @keyword
+    def go_to(self, url: str, autoformat=True):
+        """Navigates the current browser window to the provided ``url``.
+
+        If the ``autoformat`` argument is enabled (default), the
+        library adds an ``https://`` prefix if no protocol is given.
+        """
+        if autoformat:
+            url = str(url).strip()
+            if not re.match(r"^\w+:", url):
+                self.logger.debug("Adding missing protocol to URL (https://)")
+                url = "https://" + url
```

Comment:
And then you can use `urllib.parse.urlunsplit(...)` over the listed parts above where you just replace the `scheme` with `https`. ([example](https://github.com/robocorp/example-oauth-email/blob/master/OAuth2.py#L16))